docs/releases/v10.2.0.rst

```diff
@@ -0,0 +1,151 @@
+
+PostgREST 10.2.0
+================
+
+This minor version adds bug fixes and some features that provide stability to v10.0.0. These release notes also include the changes added in versions `10.1.0 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.0>`_, `10.1.1 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.1>`_ and `10.1.2 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.2>`_. You can look at the detailed changelog and download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v10.2.0>`_.
+
+Features
+--------
+
+Connection Pool
+~~~~~~~~~~~~~~~
+
+To prevent memory leaks caused by long-lived connections, PostgREST limits their lifetime in the pool through the :ref:`db-pool-max-lifetime` configuration variable.
+Likewise, there is now a time limit to wait for new connections in the pool, that is, if a new request cannot get a connection in the time specified in :ref:`db-pool-acquisition-timeout` then a response with `504` status is returned
+
+Documentation improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Added HTTP status codes to the :ref:`pgrst_errors`.
+
+* Added a how-to on :ref:`sql-user-management-using-postgres-users-and-passwords`.
+
+* Updated the :ref:`Heroku installation page <deploy_heroku>`.
+
+Changes
+-------
+
+* Removed ``db-pool-timeout`` option because it was removed in the ``hasql-pool`` library that PostgREST uses for SQL connections. (`#2444 <https://github.com/PostgREST/postgrest/issues/2444>`_)
+
+* :ref:`update` that do not affect any rows no longer return a ``404`` status. (`#2343 <https://github.com/PostgREST/postgrest/issues/2343>`_)
+
+* There is now a stricter parsing of the query string. Instead of silently ignoring, the parser now returns a :ref:`PostgREST error <pgrst100>` on invalid syntax. (`#2537 <https://github.com/PostgREST/postgrest/issues/2537>`_)
```

Comment:
There should not be "Changes" on a non-major version. Both 2 and 3 here can be labeled fixes.

Now 1 should have been a breaking change, however we didn't label the version properly and that ship has sailed.

Because of that, we can leave the "Changes" section and only have that single line.